src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier.php

```diff
@@ -46,15 +46,24 @@
             ->tag('notifier.channel', ['channel' => 'browser'])
 
         ->set('notifier.channel.chat', ChatChannel::class)
-            ->args([service('chatter.transports'), service('messenger.default_bus')->ignoreOnInvalid()])
+            ->args([
+                service('chatter.transports'),
+                abstract_arg('message bus'),
```

Comment:
Is it correct with the whitespace?

Reply:
Yes, that's correct. This is the description of the abstract argument.